src/hooks.ts

```diff
@@ -177,24 +177,25 @@ export function useMMKVObject<T>(
   key: string,
   instance?: MMKV
 ): [value: T | undefined, setValue: (value: T | undefined) => void] {
-  const [string, setString] = useMMKVString(key, instance);
+  const [json, setJson] = useMMKVString(key, instance);
+
+  const value = useMemo(() => (json ? JSON.parse(json) : undefined), [json]);
```

Comment:
Can you revert the change in `useMemo`? I liked the explicit `if` better, it's more readable.